white_list_ips.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "ip": "147.139.201.164"
+    },
+    {
+        "ip": "170.103.35.27"
+    },
+    {
+        "ip": "192.168.0.3"
```

Comment:
nit: should we just commit empty file?
you already added example into the readme which is great!